reviewer.py

```diff
@@ -80,6 +96,15 @@ def get_cherry_pick_sha_from_local_sha(self, local_sha):
     # Use the last SHA found in the patch, since it's (probably) most recent
     return self.get_cherry_pick_shas_from_patch(commit_message)[-1]
 
+  def get_commit_from_patchwork(self, url):
+    regex = re.compile('https://([a-z\.]*)/([a-z/]*)/([0-9]*)/')
+    m = regex.match(url)
+    if not m or not (m.group(1) in self.PATCHWORK_WHITELIST):
+      sys.stderr.write('ERROR: URL "%s"\n' % url)
+      return None
+    cmd = ['curl', '-o-', url + 'raw/']
```

Comment:
Use an appropriate python module for this (like requests)?

Reply:
Done.